src/components/common/Loader.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <a-spin :class="loaderClassName" tip="Загрузка..."></a-spin>
+</template>
+
+<script>
+export default {
+  name: 'Loader',
+  props: {
+    className: String,
+  },
+  computed: {
+    loaderClassName() {
+      return this.className || 'loader';
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+.loader {
+  margin-top: 40vh !important;
```

Comment:
Why this style is marked as `!important`?

Reply:
@Xoxol It doesn't work without this, changes default styles